assets/style.css

```diff
@@ -62,7 +62,8 @@ p {
   width: 500px;
   height: 50px;
   text-align: center;
-  border: none;
+  border-radius: 15px;
+  border-color: #ff426b;
```

Comment:
To be honest I have to say I found the color here a bit distracting. I prefer to keep it `border: none`. But I do like the border-radius, you can keep that.